src/components/UpdatePaymentDetailsPopup/Steps/FinalizeAddPaymentDetails.js

```diff
@@ -33,6 +34,7 @@ const FinalizeAddPaymentDetails = () => {
           })
         );
         dispatch(fetchPaymentDetails());
+        dispatch(setGracePeriodError(false));
```

Comment:
was this decision (to hide the error banner temporarily,  when the payment method was updated) approved by the product team?

Reply:
Nope, will ask then